book_object.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_reader :rentals
+  attr_accessor :title, :book
```

Comment:
- [ ] [REQUIRED] Good job so far! Instance variables should have true `setters` and `getters` please remove line `2` and change `attr_accessor` values like below:

```ruby
attr_accessor :title, :author, :rentals
```